hack/build/bundle.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -e
+
+PLATFORM="${1:-openshift}"
+VERSION="${2:-0.0.1}"
+OLM_IMAGE="${3:-registry.connect.redhat.com/dynatrace/dynatrace-operator:v${VERSION}}"
+BUNDLE_CHANNELS="${4:-}"
+BUNDLE_DEFAULT_CHANNEL="${5:-}"
+
+KUSTOMIZE="$(hack/build/command.sh kustomize 2>/dev/null)"
+if [ -z "${KUSTOMIZE}" ]; then
+  echo "'kustomize' command not found"
+  exit 2
+fi
+
+OPERATOR_SDK="$(hack/build/command.sh operator-sdk 2>/dev/null)"
+if [ -z "${OPERATOR_SDK}" ]; then
+  echo "'operator-sdk' command not found"
+  exit 2
+fi
+
+SERVICE_ACCOUNTS=(
+--extra-service-accounts dynatrace-dynakube-oneagent
+--extra-service-accounts dynatrace-dynakube-oneagent-unprivileged
+--extra-service-accounts dynatrace-kubernetes-monitoring
+--extra-service-accounts dynatrace-activegate
+)
+
+function splitOperatorSdkOptions() {
+    # shellcheck disable=SC2034
+    OPERATOR_SDK_OPTIONS=("$@")
+}
+
+eval splitOperatorSdkOptions "${SERVICE_ACCOUNTS[@]}" "${BUNDLE_CHANNELS}" "${BUNDLE_DEFAULT_CHANNEL}"
```

Comment:
what kind of magic do these bash commands do?

Reply:
eval .... filters out empty values from the list of parameters.

I agree it's tricky...